### ISSUE TITLE
test_fish are failing with fish 3.3

### ISSUE THREAD

**Issue by frenzymadness** [pypa/virtualenv#2139]
Hello.

It seems that the tests of fish integration are not compatible with the latest fish release 3.3. See the full log: https://kojipkgs.fedoraproject.org/work/tasks/5565/71425565/build.log

**Comment by gaborbernat** [comment]
PRs welcome 👍

**Comment by frenzymadness** [comment]
Investigation: Usage of bash-style PATH is possible in fish 3.2:
```
# echo $FISH_VERSION 
3.2.1
# set -gx PATH "/sbin:/bin"
# echo $PATH
/sbin /bin
```
but not in 3.3:
```
# echo $FISH_VERSION
3.3.0
# set -gx PATH "/sbin:/bin"
set: Warning: $PATH entry "/sbin:/bin" is not valid (No such file or directory)
set: Did you mean 'set PATH $PATH /bin'?
# echo $PATH
/boooo /root/.local/bin /root/bin /usr/local/sbin /usr/local/bin /usr/sbin /usr/bin /sbin /bin
```
It seems to be a problem in fish 3.3.0 which should be fixed in 3.3.1: https://github.com/fish-shell/fish-shell/issues/8095
